src/curve/stable/StablePair.sol

```diff
@@ -367,6 +350,7 @@ contract StablePair is ReservoirPair {
         }
     }
 
+    // TODO: Do we need this function?
```

Comment:
only for viewing / reporting purposes to track the growth of the value of the LP token over time